guides/common/modules/proc_preparing-for-using-external-databases.adoc

```diff
@@ -0,0 +1,27 @@
+[id="Preparing_for_using_external_databases_{context}"]
+= Optional: Preparing {Project} for using external databases
+
+By default, the {Project} installation process includes installing a PostgreSQL database on the same host as {ProjectServer}.
+However, in certain {Project} deployments, using external databases instead of the default local databases can help with the server load.
+
+You can set up your {Project} deployment to use an external PostgreSQL database during the installation of {Project}. 
+You can also migrate to an external PostgreSQL database on an existing {Project} deployment.
+
+.Procedure
+
+. Consider whether using an external database is beneficial for your use case. 
+For more information, see {AdministeringDocURL}postgresql-as-an-external-database-considerations_admin[PostgreSQL as an external database considerations] in _{AdministeringDocTitle}_.
+
+. If you want to use an external external PostgreSQL database for your {Project} deployment, prepare your {Project} environment as follows:
+
+.. Set up a system to host the external database. 
+For instructions, see {AdministeringDocURL}preparing-a-host-for-external-databases_admin[Preparing a host for external databases] in _{AdministeringDocTitle}_.
+
+.. Install PostgreSQL on the external database host you prepared. 
+For instructions, see {AdministeringDocURL}installing-postgresql_admin[Installing PostgreSQL] in _{AdministeringDocTitle}_.
+
+.Next steps
+
+* To set up an external databases when installing {Project}, see xref:configuring-server_{project-context}[].
```

Comment:
I'm not entirely sure whether this line is needed. I think all users will eventually reach the xref'ed section, won't they? Do we need to link to it if they will?

Reply:
Well, what I wanted to distinguish is the options to migrate to an external database while installing Satellite, as opposed to on an already installed environment. 

Also, assuming that the reader reads any other specific part of content before/after is generally discouraged, and even if the xref might be extraneous for some readers, for others it would still be helpful that they can click the link and see the instructions.

So all in all, I'd keep it as is :-)